diagnostic.md

```diff
@@ -15,23 +15,30 @@ describes a many-to-many relationship for `Profiles`, `Movies` and `Favorites`
 join table with references to `Movies` and `Profiles`.
 
 ```sh
-  # < Your Response Here >
+Im confused. You already explained the entity relationship between everything
+for me, so Im not sure what to write.
 ```
```

Comment:
The question asks you to provide the database structure and explain the relationship.

```
profiles
- id (integer PK)
- given_name (string)
- surname (string)
- email (string)

movies
- id (integer PK)
- title (string)
- release_date (date)
- length (integer)

favorites
- id (integer PK)
- movie_id (integer FK, references movies(id))
- profile_id (integer FK, references profiles(id))
```

something like this, for example.